### ISSUE TITLE
Problem finding other installed versions of java.

### ISSUE THREAD

**Issue by kolan2222** [Kron4ek/minecraft-vortex-launcher#83]
I am using IBM Semeru JDK 11.0.20.1 x64 installed via msi package. The paths in the environment variables are correct.  
Yes, I know that it is possible to manually specify the path to java, but I decided to suggest this fix. I think it is appropriate.
You may have to scan the paths in the registry. (guessing)
And output the list of installed java from oracle versions not only for JRE but also for JDK from any developer(vendor).
![изображение](https://github.com/Kron4ek/minecraft-vortex-launcher/assets/67638604/6ea20369-4c6c-4d77-81cd-f64dd7522d80)

**Comment by SeryiBaran** [comment]
> create your own new folder, put your new .bat there, create 2 new folder____etc.

Isn't "custom java" feature easier?

**Comment by xdeadstokex** [comment]
> Isn't "custom java" feature easier?

Indeed, Well I mean to answer directly to the OP question as "why", im quite silly :D

**Comment by Brensom** [comment]
## Bug report: custom Java path is ignored  
**Launcher:** VLauncher_1.1.20_x64_Linux  
**OS:** Arch Linux x86_64 (kernel 6.17.9-arch1-1)  
**Installed JDKs:**  
- java-8-openjdk  ( `/usr/lib/jvm/java-8-openjdk` )  
- java-17-openjdk  
- java-25-openjdk  (system default)  

---

### Steps to reproduce
1. Launch `./VLauncher_1.1.20_x64_Linux`
2. Settings → Java → “Java executable” → browse to  
   `/usr/lib/jvm/java-8-openjdk/jre/bin/java`  
   (file exists, is executable, and reports version 1.8.0_422)
3. Save settings and close/re-open launcher  
   (GUI still shows the correct path after restart)
4. Click “Play” → instant popup:  
   **“Java not found. Please install Java 8 or newer.”**

---

### What happens
- Launcher **does not execute** the binary selected in the GUI  
- Instead it falls back to an **internal hard-coded list** that only contains  
  `/usr/lib/jvm/java-25-openjdk/bin/java` (system default)  
- Because that binary is **Java 25**, the embedded start-up flags  
  `-XX:+UseConcMarkSweepGC` are **rejected** → JVM aborts  
- The custom path is **never spawned**, so the error dialog is shown **before** any user-supplied java is tried

---

### Expected behaviour
If a user has explicitly chosen an executable in  
Settings → Java → “Java executable”, that binary must be used **verbatim**  
for every launch, completely bypassing any auto-detection logic.

**Comment by SeryiBaran** [comment]
Hi! I'm using Windows, so i can't reproduce your issue. Can you please:

- find `339` line in `vlauncher_linux.pb`:
  ```basic
                    RunProgram(javaBinaryPath, fullLaunchString, workingDirectory)
  ```
  and change to this:
  ```basic
                    MessageRequester("Info", "javaBinaryPath is: " + javaBinaryPath)
                    RunProgram(javaBinaryPath, fullLaunchString, workingDirectory)
  ```
- build it
- try to reproduce your issue and screenshot window with text "javaBinaryPath is: ..."
